### PR TITLE
add docker info

### PR DESCRIPTION
**Pull request by HRashidi**
Adding description about how to use docker with aana_sdk and the links to the docker config from the aana_app_template

### CODE REVIEW

File: docs/pages/docker.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+>
+> Some models use Flash Attention for better performance. You can set the build argument `INSTALL_FLASH_ATTENTION` to `true` to install Flash Attention. 
+>
+> ```bash
+> INSTALL_FLASH_ATTENTION=true docker-compose build
+> ```
+>
+> After building the image, you can use `docker-compose up` command to run the application.
+>
+> You can also set the `INSTALL_FLASH_ATTENTION` environment variable to `true` in the `docker-compose.yaml` file.
```
Comment (movchan74) [inline docs/pages/docker.md:43]: For docs use admonitions: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#admonitions

```suggestion
!!! warning

    If your applications requires GPU to run, you need to specify which GPU to use.

    The applications will detect the available GPU automatically but you need to make sure that `CUDA_VISIBLE_DEVICES` is set correctly.
    
    Sometimes `CUDA_VISIBLE_DEVICES` is set to an empty string and the application will not be able to detect the GPU. Use `unset CUDA_VISIBLE_DEVICES` to unset the variable.

    You can also set the `CUDA_VISIBLE_DEVICES` environment variable to the GPU index you want to use: `CUDA_VISIBLE_DEVICES=0 docker-compose up`.


!!! Tip

    Some models use Flash Attention for better performance. You can set the build argument `INSTALL_FLASH_ATTENTION` to `true` to install Flash Attention. 

    ```bash
    INSTALL_FLASH_ATTENTION=true docker-compose build
    ```

    After building the image, you can use `docker-compose up` command to run the application.

    You can also set the `INSTALL_FLASH_ATTENTION` environment variable to `true` in the `docker-compose.yaml` file.

```

**Review by movchan74 — CHANGES_REQUESTED** [review]

File: docs/pages/docker.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
 # Run with Docker
 
-TODO: Add instructions on how to run Aana Apps with Docker.
\ No newline at end of file
+We provide a docker-compose configuration to run the application in a Docker container in [Aana App Template](https://github.com/mobiusml/aana_app_template).
```
[the remark this answers is not in the document]
Comment (HRashidi) [reply docs/pages/docker.md]: done

File: docs/pages/docker.md [hunk 3]
```diff
@@ -1,3 +1,45 @@
...
\ No newline at end of file
+We provide a docker-compose configuration to run the application in a Docker container in [Aana App Template](https://github.com/mobiusml/aana_app_template/?tab=readme-ov-file#running-with-docker).
+
+Requirements:
+
+- Docker Engine >= 26.1.0
+- Docker Compose >= 1.29.2
+- NVIDIA Driver >= 525.60.13
+
+You can edit the [Dockerfile](https://github.com/mobiusml/aana_app_template/Dockerfile.yaml) to assemble the image as you desire and
```
Comment (movchan74) [inline docs/pages/docker.md:11]: I don't think that link will work. It should have `blob/main` like https://github.com/mobiusml/aana_app_template/blob/main/pyproject.toml

**Review by movchan74 — CHANGES_REQUESTED** [review]

File: docs/pages/docker.md (hunk 3 above)
Comment (HRashidi) [reply docs/pages/docker.md]: fixed all of them

**Review by movchan74 — APPROVED** [review]
👍

**Pull request merged**